optimizely/odp/lru_cache.py

```diff
@@ -127,7 +118,3 @@ def lookup(self, key: K) -> Optional[V]:
     def save(self, key: K, value: V) -> None:
```

Comment:
Let's change (K, V) to concrete types. Generic will be good for us, but clients will define with fixed types: "String -> [String]".
```
    func lookup(key: String) -> [String]? 
    func save(key: String, value: [String]) 
    func reset() 
```